.github/workflows/ci.yml

```diff
@@ -153,6 +153,28 @@ jobs:
         if: matrix.os == ''
         run: sudo apt-get install llvm-13-dev && cargo build
 
+  bazel:
+    name: Bazel
+    runs-on: ubuntu-latest
+    # cxx repo disables to run the bazel job due to GitHub Actions's remote issue.
+    # It might be better to disable it here as well.
+    # See: https://github.com/dtolnay/cxx/commit/42c1ee806f186da58d1bddc8efea7d071a1c54af
+    #
+    # if: github.event_name != 'pull_request'
+    timeout-minutes: 45
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Bazel
+        run: |
```

Comment:
You should use `.bazelversion`. Bazelisk is already installed on the Ubuntu image. You don't need to install anything.

Bazel 6.0.0 is old. A newer version should be used.